netbout-web/src/test/java/com/netbout/rest/bout/TkIndexTest.java

```diff
@@ -87,5 +88,50 @@ public void rendersBoutPage() throws Exception {
             )
         );
     }
+    /**
+     * TkIndex can search bout messages.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void searchesMessages() throws Exception {
+        final MkBase base = new MkBase();
+        final String urn = "urn:test:99";
+        final User user = base.user(new URN(urn));
+        user.aliases().add("test-user");
+        final Alias alias = user.aliases().iterate().iterator().next();
+        final Bout bout = alias.inbox().bout(alias.inbox().start());
+        bout.messages().post("test1");
+        bout.messages().post("test2");
+        bout.messages().post("fest");
+        bout.friends().invite(alias.name());
+        MatcherAssert.assertThat(
+            XhtmlMatchers.xhtml(
+                new RsPrint(
+                    new FkBout(".*", new TkIndex(base)).route(
+                        new RqWithAuth(
+                            urn,
+                            new RqFake(
+                                "GET",
+                                String.format(
+                                    "/b/%d/search?q=test",
+                                    bout.number()
+                                )
+                            )
+                        )
+                    ).get()
+                ).printBody()
+            ),
+            XhtmlMatchers.hasXPaths(
+                "/page/bout[number=1]",
```

Comment:
@erimerturk do we really need to place the non-message paths here? I'd go with just the 2 expected results (if the matcher accepts it, of course).